app/assets/stylesheets/content/_diff.sass

```diff
@@ -45,3 +45,4 @@ ins
   background-color: #f6f6f6
   color: #505050
   border: 1px solid #e4e4e4
+  line-height: 1.75rem
```

Comment:
Originally I thought a larger line height would make the diff more readable, but I tested the normal size and you're right, it works fine. So I changed it now.